src/lib/es2022.intl.d.ts

```diff
@@ -32,13 +44,20 @@ declare namespace Intl {
         [Symbol.iterator](): SegmentIterator<T>;
     }
 
+    /**
+     * A `Segments` object is an iterable collection of the segments of a text string. It is returned by a call to the `segment()` method of an `Intl.Segmenter` object.
+     *
+     * [MDN](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Intl/Segmenter/segment/Segments)
+     */
     interface Segments {
         /**
          * Returns an object describing the segment in the original string that includes the code unit at a specified index.
          *
+         * [MDN](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Intl/Segmenter/segment/Segments/containing)
+         *
          * @param codeUnitIndex - A number specifying the index of the code unit in the original input string. If the value is omitted, it defaults to `0`.
          */
-        containing(codeUnitIndex?: number): SegmentData;
+        containing(codeUnitIndex?: number): SegmentData | undefined;

```

Comment:
It's unfortunate because this would ideally be tied to `noUncheckedIndexedAccess`